File: fix_webhook_path.py
import sqlite3, json
conn = sqlite3.connect('C:/Users/User/Downloads/Waha N&N/n8n/data/database.sqlite')
cursor = conn.cursor()
cursor.execute('SELECT nodes FROM workflow_entity WHERE name LIKE ?', ('%CRM%',))
row = cursor.fetchone()
nodes = json.loads(row[0])
for n in nodes:
    if n['type'] == 'n8n-nodes-base.webhook':
        print('Current webhook path:', n['parameters'].get('path'))
        # Fix: path should be just "crm-chat" not "webhook/crm-chat"
        n['parameters']['path'] = 'crm-chat'
        print('Fixed webhook path:', n['parameters'].get('path'))
cursor.execute('UPDATE workflow_entity SET nodes = ? WHERE name LIKE ?', (json.dumps(nodes), '%CRM%'))
conn.commit()
print('Saved!')

File: fix_webhook_path.js
const { DatabaseSync } = require('node:sqlite');
const db = new DatabaseSync('n8n/data/database.sqlite');
const WF_ID = 'WFCRM001chat01';

// Fix the webhook_entity path to match what n8n expects
const CORRECT_PATH = 'WFCRM001chat01/CRM Webhook/crm-chat';

db.prepare("UPDATE webhook_entity SET webhookPath=? WHERE workflowId=?").run(CORRECT_PATH, WF_ID);

const wh = db.prepare('SELECT * FROM webhook_entity').all();
console.log('Webhooks:', JSON.stringify(wh, null, 2));
db.close();
console.log('Fixed webhook path. Restart n8n now.');
